plugin/trino-google-sheets/pom.xml

```diff
@@ -159,6 +159,12 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <scope>runtime</scope>
```

Comment:
Doesn't any of google-sheets plugin dependency doesn't expose it explicitly ?

Reply:
Maven 4.0 has a fully transitive resolver but it doesn't seem to pick this dependency. Maybe it's excluded elsewhere

Reply:
Anyway, please review, it works on both 3.9 and 4.0.0-rc1